sdk/ml/azure-ai-ml/tests/automl_job/e2etests/test_automl_image_classification_multilabel.py

```diff
@@ -109,85 +102,74 @@ def test_image_classification_multilabel_run(
             target_column_name="label",
             primary_metric="iou",
             # These are temporal properties needed in Private Preview
-            properties=properties,
+            properties=get_automl_job_properties(),
         )
 
-        # Configure regular sweep job
+        # Configure sweep job
         image_classification_multilabel_job_sweep = copy.deepcopy(image_classification_multilabel_job)
         image_classification_multilabel_job_sweep.set_training_parameters(early_stopping=True, evaluation_frequency=1)
-        image_classification_multilabel_job_sweep.extend_search_space(
-            [
-                SearchSpace(
-                    model_name=Choice(["vitb16r224"]),
-                    learning_rate=Uniform(0.005, 0.05),
-                    number_of_epochs=Choice([1]),
-                    gradient_accumulation_step=Choice([1, 2]),
-                ),
-                SearchSpace(
-                    model_name=Choice(["seresnext"]),
-                    learning_rate=Uniform(0.005, 0.05),
-                    # model-specific, valid_resize_size should be larger or equal than valid_crop_size
-                    validation_resize_size=Choice([288, 320, 352]),
-                    validation_crop_size=Choice([224, 256]),  # model-specific
-                    training_crop_size=Choice([224, 256]),  # model-specific
-                    number_of_epochs=Choice([1]),
-                ),
-            ]
-        )
-        image_classification_multilabel_job_sweep.set_limits(max_trials=1, max_concurrent_trials=1)
-        image_classification_multilabel_job_sweep.set_sweep(
-            sampling_algorithm="Random",
-            early_termination=BanditPolicy(evaluation_interval=2, slack_factor=0.2, delay_evaluation=6),
-        )
+        image_classification_multilabel_job_sweep.set_limits(max_trials=2, max_concurrent_trials=2)
```

Comment:
Can you please specify the change you are referencing? Do you mean sampling algorithm?

Reply:
Sure. The set_limits() call is changed from (1, 1) to (2, 2) for non-component sweep.

Reply:
1. we need 2 trials to sweep over both runtime and components
2. I felt like 2 trials is a better test than 1 trial, and won't take up much time because all trials are 1 epoch.
I can switch the runtime sweep to 1 trial if that is preferable

Reply:
Thanks Rehaan. Agree with 1 and 2. If it doesn't take too long, it should be okay to leave it as 2 trials. I missed the part where number_of_epochs is set to 1 for runtime sweep too.